actionpack/lib/action_dispatch/http/content_security_policy.rb

```diff
@@ -202,6 +249,19 @@ def require_sri_for(*types)
       end
     end
 
+
+    # Specify whether a sandbox should be enabled for the requested resource
```

Comment:
```suggestion
    # Specify whether a sandbox should be enabled for the requested resource:
```

Reply:
Thanks! I've also added a link to `sandbox` and `report-uri`.